chrome/content/utils/accounts.js

```diff
@@ -88,7 +88,7 @@ var gmanager_Accounts = new function() {
   this.openMessage = function(aEmail, aLocation, aMessageId) {
     if (this._manager.isAccount(aEmail)) {
       var account = this._manager.getAccount(aEmail);
-      var message = account.getMessage(null, aMessageId);
+      var message = account.getMessage(account.password, aMessageId);
```

Comment:
Don't get it. Could you explain? 

I need to pass the password to the `_getServiceURI` funciton as this._password was null for me.